dev/NumberBox/NumberBox.cpp

```diff
@@ -195,9 +193,19 @@ void NumberBox::ValidateInput()
         {
             // Setting NumberFormatter to something that isn't an INumberParser will throw an exception, so this should be safe
             const auto numberParser = NumberFormatter().as<winrt::INumberParser>();
-            const auto parsedNum = numberParser.ParseDouble(text);
 
-            if (!parsedNum)
+            winrt::IReference<double> value;
+
+            if (AcceptsCalculation())
+            {
+                value = NumberBoxParser::Compute(textBox.Text(), numberParser);
```

Comment:
>Compute [](start = 41, length = 7)

Why is Compute a static which  takes a reference to a numberParser, why not just be a function like ParseDouble.

Reply:
ParseDouble is a method on INumberParser, whereas Compute is my method where I need to parse the numbers out of the expression.